src/services/transcript.ts

```diff
@@ -141,3 +147,6 @@ const transcriptService = {
 };
 
 export default transcriptService;
+
+const getJobTitle = (experience: MembershipHistory | StudentEmployment) =>
```

Comment:
The name of this function is misleading. Maybe `getName` would be clearer?
```suggestion
const getName = (experience: MembershipHistory | StudentEmployment) =>
```